Project/Assets/ML-Agents/Examples/FoodCollector/Scenes/GridFoodCollector.unity

```diff
@@ -171,148 +171,6 @@ PrefabInstance:
       objectReference: {fileID: 0}
     m_RemovedComponents: []
   m_SourcePrefab: {fileID: 100100000, guid: b5339e4b990ade14f992aadf3bf8591b, type: 3}
---- !u!1 &273651478
-GameObject:
```

Comment:
I copied and pasted the working GameObjects from the FoodCollector scene (since that's probably where they came from originally).